FashionConnect_api/routes/users.js

```diff
@@ -195,4 +193,79 @@ router.delete('/collections/:collectionID', async (req, res) => {
   }
 });
 
+//Route to handle forgot password
+router.post('/forgotpassword/', async (req, res) => {
+
+  try {
+    const { email, token } = req.body;
```

Comment:
You should be generating the token in your code here, not a string that comes from backend. Because then anybody if they know this endpoint can send a random token to you and then try to reset the password.